index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
Not a fan of `op.moduleName`, because there's the implicit connection to the entire previous expression. Having `op.operator` and `op.operands` makes most sense, and we parse `a + b - c` to `{operator: '-', operands: [{operator: '+', operands: ['a', 'b']}, 'c']}`. ASTz